cosmoz-image-viewer.html

```diff
@@ -594,123 +549,153 @@ <h2>An error occurred while loading the image.</h2>
 			</div>
 			<script>
 				/*eslint-env browser*/
-				const ciw = {
-					actions: {
+				(() => {
+					class CIV {
+						constructor(images = []) {
+							this._images = images;
```

Comment:
Why not use the `images` setter (after elements are inited) ?

Reply:
Yes, that is what we are anyways doing.